trade-finance-manager-api/src/v1/controllers/user/helpers/mapUserData.helper.js

```diff
@@ -1,12 +1,12 @@
 const mapUserData = (user) => ({
   username: user.username,
+  azureOid: user.azureOid,
```

Comment:
There should be no reason to have azureOid -- To look into and remove as we should not be returning to FE I would imagine